.github/ISSUE_TEMPLATE/quarterly_updates.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Quarterly Update
+about: Check-list for integrating the quarterly updates into PUDL
+title: ""
```

Comment:
Do we want the title to be something like "Quarterly updates: 2024-Q3"? If so, we can use [the date filter](https://github.com/JasonEtco/create-an-issue?tab=readme-ov-file#dates).